collamat_kareen/Activity3/style.css

```diff
@@ -0,0 +1,67 @@
+body{
+	color: #473C38;
+	font-family: Open Sans;
+}
+a{
```

Comment:
avoid global tag css declaration
please use class instead

Reply:
Noted, sir. Thank you. Updated.